src/core/best-practices.md

```diff
@@ -155,3 +155,60 @@ You can use our [Annotation Plugin](/core/software/pupil-capture/#annotations) t
 the recording on desired events, such as trigger events. Annotations can be sent via the keyboard or programmatically. 
 [This script](https://github.com/pupil-labs/pupil-helpers/blob/master/python/remote_annotations.py) demonstrates how you 
 can send remote annotations over the network.
+
+## Pupillometry
+
+[Pupillometry](https://doi.org/10.1002/wcs.1323) is the study of temporal changes in pupil diameter in response to 
+external light stimuli and/or cognitive processing. Pupil Core reports pupil diameter in mm provided by the 
+[pye3d](/developer/core/pye3d/#pye3d-pupil-detection) model: `diameter_3d`, and in pixels as observed in the eye 
+videos: `diameter`. 
+
+Pupil size in pixels is dependent on the camera to pupil distance, and is not corrected for perspective. 
+The output of pye3d more accurately reflects pupil size and will thus be preferable for most users.
+
+### Camera to Pupil Distances 
+Both eyes are measured independently and [adjusting the eye cameras](/core/hardware/#headset-adjustments/) such that 
+they are of equal distance to the eyes is important if data from both eyes are used.
+- `diameter_3d` - The global scale of each radius depends on the quality of the fit of the respective pye3d eye model 
+  (see below). If the eye model is estimated too far away from the eye camera for one or both of the eyes, it will lead 
+  to an overestimation of the pupil radius; if the model is estimated too close to the eye camera, it will lead to an 
+  underestimation of the pupil radius.
+- `diameter` - Measured in pixels and thus dependent on the camera to pupil distance. Pupil size estimates
+  will vary if unequal for both eyes
+
+### pye3d model
+A well-fit [pye3d](/developer/core/pye3d/#pye3d-pupil-detection) model is important for accurate estimates of pupil size 
+in mm. To generate a well-fitting model, sample sufficient gaze points from a variety of gaze angles, e.g. by moving the 
+head around while looking at a fixed position. A well-fit model is visualized by a stable circle that surrounds the 
+modelled eyeball, and this should be of an equivalent size for each eye. A dark blue circle indicates that the model is 
+within physiological bounds, and a light blue circle out of physiological bounds.
+
+<v-img :src="require('../media/core/imgs/bp-pye3d.png')"></v-img>
+<br>
+
+See the [pye3d release notes](https://github.com/pupil-labs/pupil/releases/tag/v3.4)for further details.
+
+### Freeze the pye3d model
+The [pye3d model](/developer/core/pye3d/#pye3d-pupil-detection) regularly updates to account for headset slippage
+(erroneous movements of the headset on the wearer). Abrupt changes to the calculated 3d eyeball position (due to errors 
+in the estimation) can lead to a false change in pupil size estimation. This can occur even if the actual pupil size was 
+constant. One solution to prevent this is to freeze the model (available as a toggle in the eye windows).
```

Comment:
Maybe we should clarify that there are two possibilities for incorrectly measured pupil size changes (assuming constant pupil size):
- hard slippage: the model has not adjusted yet, while the 2d pupil datum is generated from the slipped headset position
- model adjustment: no slippage happened but the model adjusted

Freezing the model only fixes the latter. The former cannot be controlled via software (unless we had frame-by-frame 3d pupil detection, which we don't).